src/System.Threading.AccessControl/src/System.Threading.AccessControl.csproj

```diff
@@ -17,10 +26,15 @@
   <ItemGroup Condition="'$(TargetsNetFx)' == 'true'">
     <Reference Include="mscorlib" />
     <Reference Include="System" />
+    <Compile Include="System\Threading\EventWaitHandleAcl.net46.cs" />
     <Compile Include="System\Threading\ThreadingAclExtensions.net46.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsNetFx)' != 'true'">
+    <Reference Include="System.Memory" />
     <Reference Include="System.Security.AccessControl" />
     <Reference Include="System.Security.Principal.Windows" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="System\Threading\MutexAcl.cs" />
```

Comment:
Build failing because this file is in the wrong location in the csproj. Fixing...